homeassistant/components/sensor/google_travel_time.py

```diff
@@ -65,6 +68,8 @@
         }))
 })
 
+TRACKABLE_DOMAINS = ["device_tracker", "sensor", "zone"]
```

Comment:
Add a `.` after each domain. Otherwise you will match a future component that is called `sensor_hello.object`